### PR TITLE
Add link to FINOS InnerSource SIG

### LINKED ISSUE #5: Link to FINOS InnerSource Special Interest Group (SIG)?
I just found [resources.md](https://github.com/finos/InnerSource/blob/master/docs/resources.md) by the FINOS InnerSource Special Interest Group (SIG).

Looks like they also started collecting awesome resources about InnerSource.
However they came roughly as far as we are in this repo so far :)

Mostly they are linking to resources from the ISC anyways.

So maybe the only thing for us to do for the moment would be to link to the [repo](https://github.com/finos/InnerSource) to say "there is such a think as the FINOS InnerSource SIG"?

@claredillon or @clcoffey does the FINOS InnerSource SIG already have a website where they publish InnerSource-related content? Or is it just this repo for now?

### PR DESCRIPTION
**Pull request by spier**
Closes #5 

I did not add this new subsection to the ToC, as I wasn't 100% sure how to do that.

### CODE REVIEW

**Comment by spier** [comment]
Nice, the linter is also complaining that the ToC is incomplete :)
> ToC missing item for "Other Communities"  remark-lint:awesome-toc

File: README.md [hunk 1]
```diff
@@ -59,3 +59,7 @@ This [awesome](https://github.com/sindresorhus/awesome) repository is maintained
 
 <!--lint ignore double-link-->
 - https://innersourcecommons.org/stories/
+
+## Other Communities
+
+- FINOS InnerSource Special Interest Group: [website](https://innersource.finos.org), [GitHub](https://github.com/finos) - a community of people implementing, or interested in implementing, InnerSource within their financial services organization in order to increase collaboration and remove/deal with excessive ownership issues that can stall innovation.
```
Comment (pmalirz) [inline README.md]: Maybe like this to make the title a link (like in all the above items):
[FINOS InnerSource Special Interest Group](https://innersource.finos.org) - a community of people implementing, or interested in implementing, InnerSource within their financial services organization in order to increase collaboration and remove/deal with excessive ownership issues that can stall innovation.  [GitHub](https://github.com/finos).

Comment (spier) [reply README.md]:
```suggestion
- [FINOS InnerSource Special Interest Group](https://innersource.finos.org) - a community of people implementing, or interested in implementing, InnerSource within their financial services organization in order to increase collaboration and remove/deal with excessive ownership issues that can stall innovation. [GitHub](https://github.com/finos).
```

**Comment by spier** [comment]
@pmalirz I made the fixes suggested by you, and by CI :) This is ready for merging now.

**Comment by pmalirz** [comment]
Great! @spier could you pls mention me as a reviewer of this PR? I would like to see if this will appear on my mobile version of github. Thanks!

**Comment by spier** [comment]
hmm. you were already set as reviewer @pmalirz.

**Comment by pmalirz** [comment]
Ok, thx for the confirmation. I wasn't sure browsing github via Android.

**Review by pmalirz — APPROVED** [review]
All good!

**Pull request merged**